src/gentropy/config.py

```diff
@@ -169,6 +175,7 @@ class FinngenFinemappingConfig(StepConfig):
     _target_: str = (
         "gentropy.finngen_finemapping_ingestion.FinnGenFinemappingIngestionStep"
     )
+    finngen_finemapping_lead_pvalue_threshold: float = 1e-5
```

Comment:
There's one thing I'm not sure about. You have added `finngen_finemapping_lead_pvalue_threshold` to the relevant config, and refer to as `pvalue_cutoff=FinngenFinemappingConfig().finngen_finemapping_lead_pvalue_threshold` in the step. However, `finngen_finemapping_lead_pvalue_threshold` it not an argument for `FinnGenFinemappingIngestionStep`. Is it OK? Would all parameters in the config passed to the step? Would that cause any problem? @project-defiant , what do you think?

Reply:
OK, double-checked with @project-defiant and @d0choa and all parameters in the setepConfig classes needs to be parameters in the init function of the step.